config/ingredients.csv

```diff
@@ -180,5 +216,6 @@ islay_scotch, none
 creme_de_cassis, none
 placeholder_fig_brandy, none
 creme_de_violette, none
-
-
+placeholder_red_wine, none
+placeholder_white_wine, none
+angostura_cacao_bitters, none
```

Comment:
We could *maybe* use the "none" location to proc a warning that we are out of whatever ingredient, but if we do that I want to be able to edit locations from the site or something